versionedRouter.js

```diff
@@ -719,7 +719,15 @@ router.get("/transformerBuildVersion", ctx => {
 });
 
 router.get("/health", ctx => {
-  ctx.body = "OK";
+  const {
+    git_commit_sha: gitCommitSha,
+    transformer_build_version: imageVersion
```

Comment:
```suggestion
   imageVersion:  transformer_build_version
```
Wont this be reversed? As `transformer_build_version` is a env variable?

Reply:
We are just aliasing the env(`transformer_build_version`) as `imageVersion`